pages/portfolio.html

```diff
@@ -4,9 +4,81 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title class="title">My Portfolio</title>
+  <link rel="stylesheet" type="text/css" href="/stylesheets/styles.css" />
 </head>
 <body>
-  
+  <nav class="nav-block">
+    <ul class="main-nav">
+      <li>
+        <a href="index.html">Home</a>
+      </li>
+      <li>
+        <a href="about.html">About</a>
+      </li>
+      <li>
+        <a href="portfolio.html">My Portfolio</a>
+      </li>
+      <li class="right-side">
+        <a href="index.html">Kristin Lee</a>
+      </li>
+
+    </ul>
+  </nav>
+
+  <header>
+    <section class="header-cover" id="top">
+      <h2 class="cover-slogan">Kristin's Portfolio</h2>
+      <h3 class="cover-desc">Check out my projects below. Enjoy!</h3>
+    </section>
+  </header>
+
+  <section class="content-wrapper">
+    <div class="portfolio-items-wrapper">
+      <div class="portfolio-item-wrapper">
+        <img class="portfolio-img-background" src="../images/portfolio-images/viewing-party.jpeg" alt="laptop"/>
+        <div class="portfolio-button-wrapper">
+          <a href="https://github.com/Kbhlee2121/viewing-party"><button class="home-button">Viewing Party</button></a>
```

Comment:
A `button` element should not be a descendant of the `a` element (an html validator helped me catch this). We can include the class `home-button` in the `a` tag and use the class to apply styles to make it look like a button. Sooon, when we learn javascript in the browser, we'll see that we can give `button` `onclick` behavior to make it link to another page.